### ISSUE TITLE
Unable to receive responses in chat scenario

### ISSUE THREAD

**Issue by suma-sai-paluri** [Azure-Samples/azure-search-openai-demo#746]
> Please provide us with the following information:
> 

```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>I have deployed web applications successfully using [7794ba1](url) and [22a45dc
](url)and also an earlier commit of this repo(which works absolutely fine) .But now with the above to stated commit I am unable to see any responses in the chat scenario. I am using GPT-35-turbo model version 0301 . I can view the responses from the Retrieve then read approach but  I cannot obtain responses from chat approach. Is any one facing the same problem Is there anything i should check ?
![Screenshot 2023-10-04 092554](https://github.com/Azure-Samples/azure-search-openai-demo/assets/145546506/495feb28-da16-47ad-9fe5-adec7a7ed1fc)

### Any log messages given by the failure
>

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### azd version?
> run `azd version` and copy paste here.

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pamelafox** [comment]
Please see the readme FAQ about logs on App Service and retrieve the logs for your deployed app, we can debug from there.

**Comment by suma-sai-paluri** [comment]
Hi Pamela, thank you for quick response these are the error messages:
![Screenshot 2023-10-06 153139](https://github.com/Azure-Samples/azure-search-openai-demo/assets/145546506/44ad5a28-9d8c-4d65-b077-ff8677a27fb1)
Please let me know what could have went wrong?

**Comment by pamelafox** [comment]
It says it doesn't understand how to use functions with ChatGPT, but it should if your API version is the latest. Can you check app.py to see what your api_version is set to?

https://github.com/Azure-Samples/azure-search-openai-demo/blob/ab1e4806a176b084b1980a9ee7e1c55bdac1a6d5/app/backend/app.py#L230

**Comment by suma-sai-paluri** [comment]
Yes it is set to the  openai.api_version = "2023-07-01-preview"  in the app.py and  openai.api_version = "2023-05-15" in predocs.py.

**Comment by srbalakr** [comment]
Hi @suma-sai-paluri  function call is supported in versions above 0613. You can disable function call and retry again.

**Comment by suma-sai-paluri** [comment]
Thank you @srbalakr, i will disable it. It would be of great help if you could provide me a source from which i could learn in detail about the function call

**Comment by pamelafox** [comment]
You can learn about function calling here: https://platform.openai.com/docs/guides/gpt/function-calling
